hudi-common/src/main/java/org/apache/hudi/common/model/DefaultHoodieRecordPayload.java

```diff
@@ -43,6 +43,10 @@
   public static final String METADATA_EVENT_TIME_KEY = "metadata.event_time.key";
   private Option<Object> eventTime = Option.empty();
 
+  public DefaultHoodieRecordPayload(GenericRecord record, String orderingField) {
```

Comment:
can this be not passed in the properties sent to the payload? Love to not introduce a new constructor, if we don't have to